instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/LogsUtils.kt

```diff
@@ -18,18 +18,18 @@ fun initializeLogger() = Logger.Builder()
 fun Logger.sendRandomLog(
     testMethodName: String,
     forge: Forge,
-    minLogLevel: Int = 1,
-    maxLogLevel: Int = 6
+    minLogLevel: Int = 2,
+    maxLogLevel: Int = 7
 ) {
     val message = forge.anAlphabeticalString()
     val throwable = forge.aNullable { forge.aThrowable() }
     val attributes = defaultTestAttributes(testMethodName)
     when (forge.anInt(min = minLogLevel, max = maxLogLevel + 1)) {
-        1 -> v(message, throwable, attributes)
-        2 -> d(message, throwable, attributes)
-        3 -> w(message, throwable, attributes)
-        4 -> e(message, throwable, attributes)
-        5 -> i(message, throwable, attributes)
-        6 -> wtf(message, throwable, attributes)
+        2 -> v(message, throwable, attributes)
+        3 -> d(message, throwable, attributes)
+        4 -> i(message, throwable, attributes)
+        5 -> w(message, throwable, attributes)
+        6 -> e(message, throwable, attributes)
+        7 -> wtf(message, throwable, attributes)
```

Comment:
Maybe we should reference the Log values

```
import android.util.Log
```
…

```suggestion
        Log.VERBOSE -> v(message, throwable, attributes)
        Log.DEBUG -> d(message, throwable, attributes)
        Log.INFO -> i(message, throwable, attributes)
        Log.WARN -> w(message, throwable, attributes)
        Log.ERROR -> e(message, throwable, attributes)
        Log.ASSERT -> wtf(message, throwable, attributes)
```

Reply:
Indeed, it improves the readability. Thanks for the suggestion.